src/domain/collaboration/new-callout/CreateCallout/CalloutForm.tsx

```
@@ -0,0 +1,186 @@
+              references={formikState.values.framing.profile.references}
+              temporaryLocation={temporaryLocation}
+              fullWidth
+            />
+            <CalloutFormContributionSettings />
+            {/*
+            <PostTemplateSelector name="postDescription" />
+            <WhiteboardTemplateSelector name="whiteboardContent" />
+            <FormikSwitch name="opened" title={t('callout.state-permission')} />
+             */}
+          </Gutters>
+          {typeof children === 'function' ? (children as Function)(formikState) : children}
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)